cli/src/main.rs

```diff
@@ -64,7 +64,11 @@ fn main() -> Result<()> {
 
             index.write_all(NOTE)?;
 
-            for file in metadata.iter().map(|x| &x.1[0].export_path) {
```

Comment:
This was only adding `export * from 'file'` for the first metadata entry, therefore, eliminating the naming collision from TypeScript's knowledge. But when there is a naming collision we want the index.ts file to be invalid